RecoEgamma/EgammaElectronProducers/plugins/ElectronSeedProducer.cc

```diff
@@ -94,6 +107,9 @@ ElectronSeedProducer::ElectronSeedProducer( const edm::ParameterSet& iConfig )
   matcher_ = new ElectronSeedGenerator(conf_,esg_tokens) ;
 
   //  get collections from config'
+  ebRecHitCollection_ = consumes<EcalRecHitCollection> (iConfig.getParameter<edm::InputTag>("ebRecHitCollection"));
+  eeRecHitCollection_ = consumes<EcalRecHitCollection> (iConfig.getParameter<edm::InputTag>("eeRecHitCollection"));
```

Comment:
I think you should add an "if (applySigmaIEtaIEtaCut_ == true)" around these consumes statements.